### DISCUSSION TITLE
Fake-IP 模式下如何设置让本地设备不经过内核？

### DISCUSSION THREAD

**Discussion by mqiancheng** [vernesong/OpenClash#3513] [Q&A]
Fake-IP 模式下如何设置让本地设备不经过内核？例如不让我内网IP 192.168.6.123设备不走openclash内核

**Comment by duetg** [comment]
在DNS设置中选择使用防火墙转发，这样在黑白名单里就出现了局域网访问有关的控制选项。在不走代理的局域网设备IP里选择或者输入192.168.6.123就可以了

**Reply by Nothend** [reply]
> 在DNS设置中选择使用防火墙转发，这样在黑白名单里就出现了局域网访问有关的控制选项。在不走代理的局域网设备IP里选择或者输入192.168.6.123就可以了

按照兄弟的设置了，非常感谢。就是想请教下：`使用 Dnsmasq 转发`和`使用防火墙转发` 有什么区别吗？谢谢指教！
